crates/task-impls/src/helpers.rs

```diff
@@ -699,9 +699,7 @@ pub(crate) async fn update_view<TYPES: NodeType>(
     debug!("Updating view from {} to {}", *old_view, *new_view);
 
     if *old_view / 100 != *new_view / 100 {
-        // TODO (https://github.com/EspressoSystems/HotShot/issues/2296):
-        // switch to info! when INFO logs become less cluttered
-        error!("Progress: entered view {:>6}", *new_view);
+        info!("Progress: entered view {:>6}", *new_view);
```

Comment:
Willing to swap this one back if we think it's relevant. My logic is that we pretty much can search for it anyway in all of our log ingestion sources, so it feels silly to place it on the error log.